docs/guides/create-app/Export.md

```diff
@@ -2,11 +2,32 @@
 
 We follow CommonJS everywhere, and expect you to expose your App's instance in root `index.js` file:
 
+We have previously our own app in `app/index.js`. In case you've forgotten, this is what we have done.
```

Comment:
this directory is written in the form of a sequential guide. since the App class is written already two pages before, we don't need to copy/paste the same code here again.